DOCKER_SETUP.md

```diff
@@ -0,0 +1,244 @@
+# F3 Nation Slack Bot - Docker Development Setup
+
+This setup includes automatic database schema creation using the F3-Data-Models repository.
+
+## Quick Start
+
+0. For now.. Ensure apparmor or selinux is disabled on your devel box
+```
+# ubuntu / debian
+sudo apparmor_status
+# fedora / rhel
+getenforce
+```
+
+
+1. **Create your `.env` file** with your Slack credentials:
+   ```bash
+   cp .env.example .env  # if available
+   # Edit .env with your values
+   ```
+   
+   ```bash
+   # Database Configuration  
+   export DATABASE_USER=postgres
+   export DATABASE_PASSWORD=postgres
+   export DATABASE_SCHEMA=f3_schema
+   export POSTGRES_PASSWORD=$DATABASE_PASSWORD
+   export DATABASE_SCHEMA=f3_schema
```

Comment:
This one is duplicated

Reply:
ah I see line duplicate on line 28, thanks